src/main/java/pages/ShoppingCartPage.java

```diff
@@ -9,18 +9,37 @@
 @Getter
 public class ShoppingCartPage extends PageObject {
 
-    private String pageUrl = "https://demo.nopcommerce.com/cart";
+    @FindBy(css = ".remove-from-cart input")
+    private WebElement removeCheckBox;
+
+    @FindBy(css = ".qty-input")
+    private WebElement quantityField;
+
+    @FindBy(css = ".update-cart-button")
+    private WebElement updateShoppingCartButton;
+
+    @FindBy(css = "a.product-name")
+    private WebElement productName;
+
+    @FindBy(css = ".bar-notification.success")
+    private WebElement alertMessage;
 
-    @FindBy(xpath = "//input[@id='removefromcart15404']")
-    private WebElement removeCheckboxFromCart;
+    @FindBy(css = "span.close")
+    private WebElement closeAlertMessage;
 
-    @FindBy(xpath = "//input[@id='itemquantity15404']")
-    private WebElement qtyFiled;
+    @FindBy(xpath = "//div[@class='no-data']")
+    private WebElement ShoppingCartIsEmpty;
```

Comment:
code convention, variable names begin with little letter, like shoppingCartIsEmpty

Reply:
Done.
Thank You!